src/rpcclient.rs

```diff
@@ -82,7 +82,24 @@ impl RpcClient {
         // TODO: duration from config.
         match rx.recv_timeout(Duration::from_secs(60))? {
             rpc::Output::Success(ok) => Ok(serde_json::from_value(ok.result)?),
-            rpc::Output::Failure(err) => bail!("Error: {:?}", err),
+            rpc::Output::Failure(err) => {
+                // NOTE: Errors with code -32801 correspond to the protocol's ContentModified error,
+                // which we don't want to show to the user and should ignore, as the result of the
+                // request that triggered this error has been invalidated by changes to the state
+                // of the server.
+                if err.error.code.code() == -32801 {
+                    let val = serde_json::from_value(Value::Null);
```

Comment:
Rather than substituting errors with nulls and pretending we're getting values back, can we move the logic to https://github.com/autozimu/LanguageClient-neovim/blob/next/src/rpchandler.rs#L12 ?

Reply:
Haven't been able to reproduce reliably as I'm not sure how to trigger that error from the server, but I think this should be done now, moved the error handling into handle_call as you suggested.